### ISSUE TITLE
Commas dismiss signature help in strings

### ISSUE THREAD

**Issue by DanielRosenwasser** [dotnet/roslyn#29669]
![signaturehelpcommasasretriggers](https://user-images.githubusercontent.com/972891/45058784-934c7800-b04e-11e8-875a-7eca2aa0be09.gif)

## TL;DR

We believe that all trigger characters should probably be considered [retrigger characters](https://github.com/dotnet/roslyn/blob/6c6855c982600a0bf154b2ac6970524eede1304d/src/Features/CSharp/Portable/SignatureHelp/ObjectCreationExpressionSignatureHelpProvider.cs#L25).

## Context

We (@jessetrinity, @sana-ajani, @amcasey and I on the TypeScript team) are trying to tackle the Visual Studio side of https://github.com/Microsoft/TypeScript/issues/1150. We noticed that the most "intuitive" implementation actually results in the undesirable behavior highlighted above. When checking to see what C# does, we saw the same behavior. Rather than diverging, we wanted to consult with Roslyn first.

## Details

**Version Used**: Latest

**Steps to Reproduce**:

1. Write a method `Foo` accepting a string
2. Trigger signature help session within a string literal being passed to `Foo`
3. Enter a comma character within the string (`,`)

**Expected Behavior**:

Signature help remains

**Actual Behavior**:

Signature help is dismissed

**Comment by CyrusNajmabadi** [comment]
is this still an issue for you @DanielRosenwasser ?

**Comment by DanielRosenwasser** [comment]
It was more of a quirk that I thought Roslyn would be interested in fixing, but I'd understand if it's not worth the time.

**Comment by jasonmalinowski** [comment]
This impacts typing a < too.
